tests/unit/rl_worker/test_worker_util.py

```diff
@@ -0,0 +1,165 @@
+import mock
+import requests
+import responses
+
+from unittest import TestCase
+
+from scripts.workers.worker_util import EvalAI_Interface, URLS
```

Comment:
reorder imports based on package name. Import on line 5 should come after line 7

Reply:
Okay, will fix that right away.

Reply:
@Ram81 Fixed :+1: